tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/ParseFile.scala

```diff
@@ -66,11 +62,20 @@ object ParseFile {
   def parseTestSet(parser: TransitionParser, parseSource: PolytreeParseSource): Unit = {
     println("Parsing test set.")
     val startTime: Long = Platform.currentTime
-    val candidateParses: Iterator[Option[PolytreeParse]] = {
-      parseSource.parseIterator map {
-        parse => parser.parse(parse.sentence)
+
+    import scala.concurrent.duration._
```

Comment:
Actually, I should say: Move all non-implicit imports. The execution context could stay here.